cmd/lotus-shed/indexes.go

```diff
@@ -23,6 +24,13 @@ import (
 	lcli "github.com/filecoin-project/lotus/cli"
 )
 
+const (
+	// same as in chain/events/index.go
+	eventExists = `SELECT MAX(id) FROM event WHERE height=? AND tipset_key=? AND tipset_key_cid=? AND emitter_addr=? AND event_index=? AND message_cid=? AND message_index=?`
+	insertEvent = `INSERT OR IGNORE INTO event(height, tipset_key, tipset_key_cid, emitter_addr, event_index, message_cid, message_index, reverted) VALUES(?, ?, ?, ?, ?, ?, ?, ?)`
+	insertEntry = `INSERT OR IGNORE INTO event_entry(event_id, indexed, flags, key, codec, value) VALUES(?, ?, ?, ?, ?, ?)`
+)
+
```

Comment:
Updated this PR to pull these in, they're identical to chain/events/index.go, so it does suggest that maybe we should consider making them public over there. It would help keep things in sync, but would likely just result in this command failing the next time someone tries it and the parameters don't match. But we mostly have that problem already anyway.